src/definition/rooms/IRoom.ts

```diff
@@ -23,4 +24,11 @@ export interface IRoom {
     customFields?: { [key: string]: any };
     parentRoom?: IRoom;
     livechatData?: { [key: string]: any };
+    source?: {
```

Comment:
This `source` property is so generic that someone who does not know Rocket.Chat's internals would not know what this is. In fact, I personally have no idea what it is referring to. What's the purpose of this? Can it be named something better and then mapped inside of Rocket.Chat?

Reply:
I agree that this name is too generic but I just decided to use the same property name as it used on the omnichannel room on Rocket.chat project as you can see on this [PR](https://github.com/RocketChat/Rocket.Chat/pull/23912)

Reply:
Right, I understand that. However, what I am saying is that we should map it to a name that actually provides meaning instead of further compounding the confusion and generics.